### PR TITLE
fix(usa-address): rename field and remove duplicate field

### PR DESCRIPTION
**Pull request by gsasouza**
#### What problem is this solving?

Rename street and number field to work with `place-components` and remove duplicate `city` field in extended mode

#### How should this be manually tested?

[Workspace](url)

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [x] Updated `CHANGELOG.md`.
- [ ] Linked this PR to a Jira story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
_ | New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

File: packages/usa/manifest.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "vendor": "vtex",
   "name": "country-data-usa",
-  "version": "0.1.1",
+  "version": "0.1.2",
```
Comment (lucasecdb) [inline packages/usa/manifest.json:4]: you should do the version bump on a separate commit, and update the changelog too

File: packages/usa/vtex.country-data-settings/configuration.json [hunk 2]
```diff
@@ -232,11 +232,11 @@
       "maxLength": 100
     },
     "street": {
-      "label": "street",
+      "label": "streetRoadAvenue",
```
Comment (lucasecdb) [inline packages/usa/vtex.country-data-settings/configuration.json:235]: why update the label only for USA? wouldn't it make more sense to only update the translation in place components for english? because this way it will generate a new i18n key that will probably be "translated" to other languages even though it is only used in english

File: packages/usa/vtex.country-data-settings/configuration.json [hunk 3]
```diff
@@ -232,11 +232,11 @@
...
       "required": true
     },
     "number": {
-      "label": "number",
+      "label": "numberOption",
```
Comment (lucasecdb) [inline packages/usa/vtex.country-data-settings/configuration.json:239]: same as above

**Review by lucasecdb — CHANGES_REQUESTED** [review]

File: packages/usa/vtex.country-data-settings/configuration.json (hunk 2 above)
Comment (gsasouza) [reply packages/usa/vtex.country-data-settings/configuration.json:235]: It's not generating a new key, USA has wrong keys for this fields, [other languages uses this new value](https://github.com/vtex-apps/countries-data/blob/main/packages/bra/vtex.country-data-settings/configuration.json#L136)

Comment (lucasecdb) [reply packages/usa/vtex.country-data-settings/configuration.json:235]: ohhh my bad then

**Review by lucasecdb — APPROVED** [review]
LGTM, remember to undo the version bump before generating the release commit :-)

**Review by lucasecdb — APPROVED** [review]

**Review by wdsrocha — APPROVED** [review]
